libs/react-helpers/src/lib/grid/cumulative-vol-cell.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import type { ICellRendererParams } from 'ag-grid-community';
+
+import { BID_COLOR, ASK_COLOR } from './vol-cell';
+
+const INTERSECT_COLOR = 'darkgray';
+
+export interface CumulativeVolProps {
+  relativeAsk?: number;
+  relativeBid?: number;
+}
+
+export interface ICumulativeVolCellProps extends ICellRendererParams {
+  value: CumulativeVolProps;
+}
+
+export const CumulativeVol = React.memo(
+  ({ relativeAsk, relativeBid }: CumulativeVolProps) => {
+    const bid = relativeBid ? (
+      <div
+        className="h-full absolute top-0 right-0"
+        style={{
+          width: `${relativeBid}%`,
+          backgroundColor:
+            relativeAsk && relativeAsk > relativeBid
+              ? INTERSECT_COLOR
+              : BID_COLOR,
+        }}
+      ></div>
```

Comment:
chore (non-blocking): Could self close some of these divs in this file
```suggestion
      />
```